source/indexes.txt

```diff
@@ -12,6 +12,28 @@ Manage Indexes
    :depth: 1
    :class: singlecol
 
+Indexes are special data structures that support the efficient 
```

Comment:
"support the efficient execution of queries in MongoDB" sounds like too much passive voice. What about:

"Indexes are special data structures that improve query (execution|performance)."